PyInstaller/loader/pyiboot01_bootstrap.py

```diff
@@ -47,10 +47,21 @@
 # Some packages behaves differently when running inside virtual environment.
 # E.g. IPython tries to append path VIRTUAL_ENV to sys.path.
 # For the frozen app we want to prevent this behavior.
+#
+# But, even though we have enough reason to delete VIRTUAL_ENV in environment,
+# we should still keep the original data for those who relies on it essentially,
+# provide another way to access it when necessary.
```

Comment:
Please write a shorter comment, e.g. "Preserve original values those applicatons which need it."